src/pages/waiting-room/cat-home-test/waiting-room.cat-home-test.page.html

```diff
@@ -0,0 +1,40 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ 'waitingRoom.title' | translate }} - {{pageState.candidateUntitledName$ | async}}</ion-title>
+    <ion-buttons end>
+      <end-test-link category="D"></end-test-link>
```

Comment:
wrong category

Reply:
instead of a TODO - it now uses the right type

Reply:
Now passes in the correct category